scripts/markdownlint

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
where is this script used?

Reply:
added a note in a new CONTRIBUTING file.